sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -231,6 +232,7 @@ public partial class EntityDataSource
     public partial class ExtractKeyPhrasesAction
     {
         public ExtractKeyPhrasesAction() { }
+        public ExtractKeyPhrasesAction(Azure.AI.TextAnalytics.TextAnalyticsRequestOptions requestOptions) { }
```

Comment:
I realized we're using the term `requestOptions` for `TextAnalyticsRequestOptions`, but only `options` for `AnalyzeSentimentOptions`. Just double checking if the difference is intentional.

Reply:
I could make them all the same with `options` if that is better. 
I felt weird saying `requestOptions` for `AnalyzeSentimentOptions` so last minute I changed it to `options`

Reply:
Both look good to me. Just wanted to make sure if it was intentional.

Reply:
I changed them all to options :p